src/features/classification/Classification.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../styles/variables';
+
+.bdl-Classification {
+    .bdl-Classification-badge {
+        border: 1px solid;
```

Comment:
I think it would help to specify a default color and include a comment that it's overridden in JS.